lib/python/bind_operators.h

```diff
@@ -83,6 +83,90 @@ void bind_astype(py::class_<T, Ignored...> &c) {
         :rtype: Union[scipp.Variable, scipp.DataArray])");
 }
 
+template <class T, class... Ignored>
+void bind_to(py::class_<T, Ignored...> &c) {
+  c.def(
+      "to",
+      [](const T &self, const py::object &dt,
+         const std::optional<ProtoUnit> unit, const bool copy) {
+        using scipp::core::dtype;
+
+        const auto src_dtype(self.dtype());
+        const auto dest_dtype(dt.is_none() ? src_dtype : scipp_dtype(dt));
+        const auto dest_units(unit.has_value() ? make_unit(unit.value())
+                                               : self.unit());
+
+        [[maybe_unused]] py::gil_scoped_release release;
+
+        bool convert_dtype_first;
+
```

Comment:
Really minor thing:
```.cpp
const auto copy_policy = copy ? scipp::CopyPolicy::Always
                              : scipp::CopyPolicy::TryAvoid);
```
And then use that below instead of repeating the ternary.